### PR TITLE
openFrameworks 0.10.0 added

### PR DESCRIPTION
**Pull request by icq4ever**
openFrameworks 0.10.0 added

### CODE REVIEW

File: docsets/openFrameworks/docset.json [hunk 1]
```diff
@@ -9,9 +9,13 @@
     "aliases": ["of"],
 
     "specific_versions": [
+		{
+			"version": "0.10.0",
+			"archive": "versions/0.10.0/openFrameworks.tgz"
```
Comment (Kapeli) [inline docsets/openFrameworks/docset.json]: You need to add an archive at this path as well.

File: docsets/openFrameworks/docset.json [hunk 2]
```diff
@@ -9,9 +9,13 @@
...
+		},
     	{
-    		"version": "0.9.8",
-    		"archive": "versions/0.9.8/openFrameworks.tgz"
+			"version": "0.9.8",
+			"archive": "versions/0.9.8/openFrameworks.tgz"
```
Comment (Kapeli) [inline docsets/openFrameworks/docset.json]: You need to add an archive at this path as well.

File: docsets/openFrameworks/docset.json (hunk 1 above)
Comment (icq4ever) [reply docsets/openFrameworks/docset.json]: ok. I copied to that path as well.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**